### DISCUSSION TITLE
SANE scanner support

### DISCUSSION THREAD

**Discussion by rekcodocker** [nextcloud/all-in-one#1214] [Ideas] [answered]
Nextcloud has a scanner option. I would love to add a scan to Nextcloud, from nextcloud itself.
The app exists, but it needs a working SANE installation. 
Is it possible to add this to a container?

**Comment by Zoey2936** [comment]
Should be possible, when #1163 is merged and released

**Comment by szaimen** [comment, marked as answer]
This will be handled with https://github.com/nextcloud/all-in-one/issues/1162

**Reply by szaimen** [reply]
This is now released with v3.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
